analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -168,133 +144,275 @@ public static void LoopWithBreak(System.Collections.Generic.IEnumerable<string>
     }
 }
 
-public interface IWithDefaultImplementation
+class NullableOfCustomTypes
 {
-    int DoSomething()
+    void Assignment(AStruct? nullable)
     {
-        int? i = null;
-        return i.Value; //FIXME Non-compliant
+        _ = nullable.Value;                                              // Compliant, unknown
```

Comment:
Aesthetics: It would be nicer to indent the comments something like -20 spaces in this method.

Reply:
Moved 19 spaces backward. 
Moving them 20 spaces would make the `;` symbol the farthest on the right and would leave no space between the semi-colon and the `//` characters (i.e. `.Value;// FN`), which in my opinion is not nice to see.

Reply:
"like -20" means roughly :)